src/ResourceManager/DataMigration/Commands.DataMigration/Cmdlets/ConnectToTargetSqlDbMiTaskCmdlet.cs

```diff
@@ -0,0 +1,46 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
```

Comment:
Please add the correct copyright file to all these files (see the top of almost all files in our directory).